README.md

```diff
@@ -254,12 +256,14 @@ $ prodder push -c prodder.yml
 ```
 
 ## TODO
+
 * Log activity as it is performed.
 * Support tracking a particular branch instead of master.
 * Support specifying the options to pass to each pg_dump form.
 * Select dumping only a subset of a seed table. (pg_dump won't do this ...)
 
 ## Previous Contributors
+
```

Comment:
Need to add an entry for yourself here.

Reply:
I think it's just a list of the internal contributors, before they open-source the gem.
The list is pretty outdated, btw. I think we should just rely on GitHub for new contributors.

![image](https://github.com/user-attachments/assets/e3ccb618-8e86-424f-8d6d-13cc78cf4a8e)

Reply:
OK...  We should remove that deprecated section then...but it can wait.